packages/element-lib/package.json

```diff
@@ -67,10 +71,13 @@
     "pouchdb-adapter-idb": "^7.1.1",
     "pouchdb-adapter-leveldb": "^7.1.1",
     "pouchdb-adapter-memory": "^7.1.1",
+    "prettier": "^2.0.5",
```

Comment:
this should be a dev dependency

Reply:
@gjgd ... why dev dependency?

Reply:
Because this dependency is for formatting code, and we don't want it to be bundled in the lib